src/layer/heatmap/HeatMapLayer.js

```diff
@@ -51,12 +51,18 @@ define([
      * @classdesc A HeatMap layer for visualising an array of measured locations.
      * @param {String} displayName This layer's display name.
      * @param {MeasuredLocation[]} measuredLocations An array of locations with measures to visualise.
+     * @param {Number} amountOfLevels Optional. If provided it specifies the amount of levels that will be generated for
+     *  this layer. If you have only one, it behaves in a similar way to the Analytical surface from the World Wind Java.
+     *  If you have multiple levels then whenever the user zooms over certain threshold, the new version of the layer
+     *  will be generated.
+     * @param {Number} opacity Optional. The final opacity of this layer. Number between 0 and 1. If the opacity isn't
```

Comment:
What is the use case for the default situation (value undefined)? I suggest having full opacity as default and allow setting a lower one via this optional property.

In addition, as it is optional, I would also have this option as a property and not an argument of the constructor.